ingestion/src/metadata/ingestion/ometa/mixins/tag_mixin.py

```diff
@@ -0,0 +1,181 @@
+    ) -> None:
+        """Method to update a tag category
+        Args:
+            category_name (str): tag category name
+            tag_category_body (TagCategory): body of the request
+        """
+        path = f"/tags/{category_name}"
+        resp = self.client.put(path=path, data=tag_category_body.json())
+        logger.info(f"Updated tag category: {resp}")
+
+    def post_create_primary_tag(
+        self, category_name: str, primary_tag_body: Tag
+    ) -> None:
+        """Method to create a primary tag within a category
+        Args:
+            category_name (str): tag category name
+            primary_tag_body (Tag): body of the Tag for the request
+        """
+        path = f"/tags/{category_name}"
+        resp = self.client.post(path=path, data=primary_tag_body.json())
+        logger.info(f"Create primary tag in category {category_name}: {resp}")
+
+    def get_primary_tag(
+        self,
+        entity: Type[T],
+        category_name: str,
+        primary_tag_fqn: str,
+        fields: Optional[List[str]] = None,
+    ) -> Optional[T]:
+        """Get primary tag information
+
+        Args:
+            entity: entity class model
+            category_name (str): category name to get info on
+            primary_tag_fqn (str): fully qualified name of the primary tag
+            fields (List): list of fields to pass with the request
+        """
+        path = f"{category_name}/{primary_tag_fqn}"
+        return self._get(entity=entity, path=path, fields=fields)
+
+    def put_update_primary_tag(
+        self,
+        category_name: str,
+        primary_tag_fqn: str,
+        primary_tag_body: Tag,
+    ) -> None:
+        """Update primary tag info
+
+        Args:
+            entity: entity class model
+            category_name (str): category name to get info on
+            primary_tag_fqn (str): fully qualified name of the primary tag
+            primary_tag_body (Tag): body of the Tag for the request
+        """
+        path = f"/tags/{category_name}/{primary_tag_fqn}"
+        resp = self.client.put(path=path, data=primary_tag_body.json())
+        logger.info(f"Updated primary tag: {resp}")
+
+    def post_create_secondary_tag(
+        self, category_name: str, primary_tag_fqn: str, secondary_tag_body: Tag
+    ) -> None:
+        """Method to create a secondary tag under a primary tag
+        Args:
+            category_name (str): tag category name
+            primary_tag_fqn (str): primary tag fully qualified name
+            secondary_tag_body (Tag): body of the Tag for the request
+        """
+        path = f"/tags/{category_name}/{primary_tag_fqn}"
+        resp = self.client.post(path=path, data=secondary_tag_body.json())
+        logger.info(
+            f"Create secondary tag in category {category_name}"
+            f"under primary tag {primary_tag_fqn}: {resp}"
+        )
+
+    # pylint: disable=too-many-arguments
```

Comment:
Is this disabling needed here? At `.pylintrc` we have `max-args=7` 🤔

Reply:
Ah I ran pylint from the repo just for this file. Might be why this got flagged. I’ll remove it.